docs/pages/setup/guides/local-docker.mdx

```diff
@@ -0,0 +1,198 @@
+
+```bash
+docker exec teleport tctl users add testuser root,ubuntu
+```
+
+When you run this command, Teleport will output a URL that you must open to complete the user signup process:
+
+```bash
+User testuser has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h0m0s:
+https://localhost:3080/web/invite/4f2718a52ce107568b191f222ba069f7
+
+NOTE: Make sure localhost:3080 points at a Teleport proxy which users can access.
+```
+
+You can now [follow this guide from **Step 4** onwards](../../getting-started.mdx#step-44-have-fun-with-teleport) to create your user and log into Teleport or continue below.
+
+## tsh into your Teleport container
+
+Finish signing up and creating your user using the generated link created previously.
+
+[Download and install](https://goteleport.com/teleport/download) a copy of Teleport locally. Doing so will install the `tsh` tool so you can interact with Docker containers.
+
+Open a second terminal and issue the command:
+
+```bash
+tsh login --proxy=localhost:3080 --insecure --user=testuser
+```
+
+<Admonition type="note" title="Note">
+  The `--insecure` flag is not recommended in production but can be used to bypass certain TLS and port requirements when testing locally.
+</Admonition>
+
+You will be prompted to enter the password and One-Time Passcode you created for your user `testuser`:
+
+```bash
+Enter password for Teleport user testuser:
+Enter your OTP token:
+9999999
+```
+
+After successfully authenticating you should see the following in your terminal:
+
+```
+WARNING: You are using insecure connection to SSH proxy https://localhost:3080
+> Profile URL:        https://localhost:3080
+  Logged in as:       testuser
+  Cluster:            localhost
+  Roles:              admin
+  Logins:             root, ubuntu
+  Kubernetes:         disabled
+  Valid until:        2021-06-10 07:15:42 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+```
+
+Running the next command will display all Teleport Nodes you're connected to:
+
+```bash
+tsh ls
+```
+
+```
+# Output
+Node Name Address        Labels                          
+--------- -------------- ------------------------------- 
+localhost 127.0.0.1:3022 env=example, hostname=localhost 
+```
+
+To SSH into the local Node `localhost` (running in your Docker container) issue the following `tsh` command:
+
+```bash
+tsh ssh root@localhost       
+```
+
+This will bring up the Linux command prompt where you can issue Bash commands, traverse the directory tree, and explore the container contents:
+
+```bash
+root@localhost:~# 
+```
+
+<Admonition type="tip" title="Tip">
+  To recreate `testuser`:
+
+  ```bash
+  # Remove the current testuser, password, add token, and OTP
+  docker exec teleport tctl users rm testuser 
+
+  # Recreate testuser    
+  docker exec teleport tctl users add testuser root,ubuntu   
+  ```
+</Admonition>
+
+## Cleanup
```

Comment:
I'd remove this section. Folks usually know how to remove things if they use docker compose. Instead of this, let's add next steps:

* Manual configuration section from above
* Point to explore Kubernetes setup guides (helm) and try other products - database access to show more value.